test/tests_for_sdkmr/EE2Runjob_test.py

```diff
@@ -381,7 +390,9 @@ def test_run_job_as_admin_with_job_requirements():
     sdkmr.check_as_admin.assert_called_once_with(JobPermissions.WRITE)
     jrr.normalize_job_reqs.assert_called_once_with(inc_reqs, "input job")
     jrr.get_requirements_type.assert_called_once_with(**req_args)
-    jrr.resolve_requirements.assert_called_once_with(_METHOD, **req_args)
+    jrr.resolve_requirements.assert_called_once_with(
+        _METHOD, sdkmr.get_catalog_cache(), **req_args
```

Comment:
I would `s/sdkmr.get_catalog_cache()/mocks[CatalogCache]/ throughout this file for the `resolve_requirements` call to remove indirection

Reply:
Done

Reply:
:+1: